igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -360,6 +518,93 @@
     "plt.tight_layout()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "0f428b9b-f494-4df2-9f06-56b3610aaf4c",
+   "metadata": {},
+   "source": [
+    "Each line in the top graph represents an interferogram. You can see that most interferograms span a relatively small date window, but that there are a few long temporal baseline interferograms that span the winter snow season. The bottom graph is a histogram displaying how many interferograms cover each date. As you can see, we have at least one interferogram for each date, so our network is fully connected!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f6b71dcf-19fc-4d6f-9f38-247f6ec4997e",
+   "metadata": {},
+   "source": [
+    "##### 3.3.1.2 Network Quality"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7318e24-3a07-4b43-964a-c1b1e820d016",
+   "metadata": {},
+   "source": [
+    "The next thing we want to consider is network quality. Setting an average coherence threshold of `0.7` already excludes a lot of interferograms (the interferograms whose numbers are red in the graph below are excluded), but there are still some we could consider removing to avoid including interferograms with unwrapping errors. For instance take a look at interferogram number 12 in the `ascending_50` stack."
```

Comment:
```suggestion
    "The next thing we want to consider is network quality. Setting an average coherence threshold of `0.7` already excludes a lot of interferograms (the interferograms whose numbers are red in the graph below are excluded), but there are still some we could consider removing to avoid including interferograms with unwrapping errors. For instance, take a look at interferogram number 12 in the `ascending_50` stack."
```